src/constants/settings.ts

```diff
@@ -1,28 +1,48 @@
-export const PAPER_APP_URL =
-	// Use `localhost:3000` on dev.
-	process?.env?.NEXT_PUBLIC_NODE_ENV === "development" ||
-	process?.env?.NODE_ENV === "development"
-		? "http://localhost:3000"
-		: // Use current window host on staging.
-		process?.env?.NEXT_PUBLIC_NODE_ENV === "staging" &&
-		  typeof window !== "undefined"
-		? window.location.origin
-		: // Use "paper.xyz" if on that domain (deprecated).
-		typeof window !== "undefined" &&
-		  window.location.origin === "https://paper.xyz"
-		? "https://paper.xyz"
-		: // Fall back to the canonical hostname "withpaper.com".
-		  "https://withpaper.com";
+const isDev = (): boolean => {
+  if (
+    process?.env?.NEXT_PUBLIC_NODE_ENV === "development" ||
+    process?.env?.NODE_ENV === "development" ||
+    (typeof window !== "undefined" &&
+      window.location.origin.includes("localhost"))
+  )
+    return true;
```

Comment:
We actually don't want this part because many devs will be testing their implementation on `localhost:*`, and this would cause the SDK to load `localhost:3000/sdk/2022-08-12/...` which will fail.

Reply:
Thanks @arcoraven , would it be an idea to introduce more explicit vars on our local dev environment rather than relying on process.env? Perhaps something on the window object only for local dev?